bin/plugin/commands/changelog.js

```diff
@@ -146,7 +146,7 @@ const LABEL_FEATURE_MAPPING = {
 	'[Type] Documentation': 'Documentation',
 	'Global Styles': 'Global Styles',
 	'[Type] Build Tooling': 'Build Tooling',
-	'npm Packages': 'Plugin',
+	'npm Packages': 'NPM Packages',
```

Comment:
Nit: it's npm not NPM :)

<img width="857" alt="Screen Shot 2021-07-23 at 10 11 28" src="https://user-images.githubusercontent.com/699132/126754553-22e08e19-e2a6-400c-aca6-96bfbcf2d445.png">